pom.xml

```diff
@@ -0,0 +1,357 @@
+			</properties>
+			<modules>
+				<module>org.eclipse.emfcloud.validation.framework</module>
+				<module>targetplatform</module>
+				<module>releng</module>
+			</modules>
+			<build>
+				<pluginManagement>
+					<plugins>
+						<plugin>
+							<groupId>org.eclipse.tycho</groupId>
+							<artifactId>tycho-compiler-plugin</artifactId>
+							<version>${tycho-version}</version>
+						</plugin>
+						<!-- enable tycho build extension -->
+						<plugin>
+							<groupId>org.eclipse.tycho</groupId>
+							<artifactId>tycho-maven-plugin</artifactId>
+							<version>${tycho-version}</version>
+							<extensions>true</extensions>
+						</plugin>
+						<plugin>
+							<!-- enable source bundle generation -->
+							<groupId>org.eclipse.tycho</groupId>
+							<artifactId>tycho-source-plugin</artifactId>
+							<version>${tycho-version}</version>
+						</plugin>
+						<plugin>
+							<groupId>org.eclipse.tycho</groupId>
+							<artifactId>target-platform-configuration</artifactId>
+							<version>${tycho-version}</version>
+						</plugin>
+					</plugins>
+				</pluginManagement>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-resources-plugin</artifactId>
+						<version>2.7</version>
+						<configuration>
+							<encoding>UTF-8</encoding>
+						</configuration>
+					</plugin>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>tycho-maven-plugin</artifactId>
+						<extensions>true</extensions>
+					</plugin>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>tycho-compiler-plugin</artifactId>
+						<configuration>
+							<source>${java.source}</source>
+							<target>${java.target}</target>
+							<optimize>true</optimize>
+							<showWarnings>true</showWarnings>
+						</configuration>
+					</plugin>
+					<plugin>
+						<!-- enable source bundle generation -->
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>tycho-source-plugin</artifactId>
+						<executions>
+							<execution>
+								<id>plugin-source</id>
+								<goals>
+									<goal>plugin-source</goal>
+								</goals>
+							</execution>
+						</executions>
+					</plugin>
+					<plugin>
+						<groupId>org.eclipse.tycho</groupId>
+						<artifactId>target-platform-configuration</artifactId>
+						<configuration>
+							<includePackedArtifacts>true</includePackedArtifacts>
+							<targetDefinitionIncludeSource>honor</targetDefinitionIncludeSource>
+							<environments>
+								<environment>
+									<os>win32</os>
+									<ws>win32</ws>
+									<arch>x86_64</arch>
+								</environment>
+								<environment>
+									<os>linux</os>
+									<ws>gtk</ws>
+									<arch>x86_64</arch>
+								</environment>
+								<environment>
+									<os>macosx</os>
+									<ws>cocoa</ws>
+									<arch>x86_64</arch>
+								</environment>
+							</environments>
+							<target>
```

Comment:
you can specify the target file directly using tycho 2.3: https://wiki.eclipse.org/Tycho/Release_Notes/2.3#Target_files_can_be_specified_directly_now

before 2.3 it is also not needed to have a classifier, the target file just needs to be named as the artifactid see https://github.com/eclipse-emfcloud/coffee-editor/tree/master/backend/releng/org.eclipse.emfcloud.coffee.target